app/static/js/vendor/jquery/jquery.codezero.js

```diff
@@ -150,13 +150,11 @@ var cachedFuzzyMatch;
  * Cache the results of a RegExp match.
  * @type {Function}
  */
-if ('undefined' === !typeof _) {
-    cachedFuzzyMatch = _.memoize(function (pattern) {
-        return new RegExp(pattern.split("").reduce(function (a, b) {
-            return a + '[^' + b + ']*' + b;
-        }), 'i');
-    });
-}
+cachedFuzzyMatch = _.memoize(function (pattern) {
```

Comment:
@magdalenesuo the if statement was intentional ... `jquery.codezero.js` is included in some pages where lodash is not included. So in those pages, an error will occur if we try to use a lodash function without the lodash library.